src/core/parser.sk

```diff
@@ -1056,7 +1062,8 @@ namespace GLSLX.Parser {
       pratt = createExpressionParser
     }
 
-    var context = ParserContext.new(log, tokens, data, resolver)
+    const processedIncludes = StringMap<bool>.new
```

Comment:
I didn't see bindings for ES6 `Set`, so I'm using a `StringMap` instead. LMK if you'd prefer something else!